apps/devhub/tasks.py

```diff
@@ -54,6 +55,25 @@ def validate(file_, listed=None):
         return result
 
 
+def validate_and_submit(addon, file_, listed=None):
+    return validate(file_, listed=listed,
+                    callback=submit_file.si(addon.pk, file_.pk))
+
+
+@task
+def submit_file(addon_pk, file_pk):
+    addon = Addon.unfiltered.get(pk=addon_pk)
+    file_ = FileUpload.objects.get(pk=file_pk)
+    validation = json.loads(file_.validation)
+    if (file_.automated_signing and validation['passed_auto_validation']) or (
+            not file_.automated_signing and file_.valid):
+        # Import loop.
+        from devhub.views import auto_sign_version
+
```

Comment:
I think we'll have enough metrics from the stats on signing itself (which is nested within the Annotator).

However, there was discussion on IRC about wanting to get a count of API usage. That would be a nice thing to add to the new API at a generic level (i.e. all API requests). I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1217143